### PR TITLE
Remind FIP SITs to add ECTs and mentors

### PR DESCRIPTION
**Pull request by jamesrose**
## Ticket and context

Ticket: https://dfedigital.atlassian.net/browse/CPDRP-994

I don't think warrants tests, but some keen eyes would be appreciated. Plan is send tomorrow morning and then revert this PR.

### CODE REVIEW

File: app/mailers/school_mailer.rb [hunk 1]
```diff
@@ -266,6 +267,20 @@ def induction_coordinator_add_participants_email(recipient:, name:, sign_in_url:
...
+
+    template_mail(
+      FIP_ADD_ECTS_AND_MENTORS_EMAIL_TEMPLATE,
+      to: induction_coordinator.user.email,
+      rails_mailer: mailer_name,
+      rails_mail_template: action_name,
+      personalisation: {
+        sign_in: new_user_session_url(**campaign_tracking),
+      },
+    ).tag(:request_to_add_ects_and_mentors).associate_with(induction_coordinator, as: :participant_profile)
```
Comment (kicferk1) [inline app/mailers/school_mailer.rb:281]: `.associate_with(induction_coordinator, as: :induction_coordinator_profile)`?

**Review by kicferk1 — APPROVED** [review]
Unit tests, yes or no?

File: app/mailers/school_mailer.rb (hunk 1 above)
Comment (jamesrose) [reply app/mailers/school_mailer.rb]: Yep, thanks!

**Comment by jamesrose** [comment]
@kicferk1 We've tried doing it and not doing it. I think it depends on how short-lived the mailer is.

**Comment by kicferk1** [comment]
Yeah, I didn't like not doing it, cause nobody actually ran the code before trying to send emails, and it crashed 😂

**Comment by jamesrose** [comment]
@kicferk1 I've ran this one :)

**Pull request merged**